### ISSUE TITLE
Order 2N3906

### ISSUE THREAD

**Issue by CoogLFish** [TOILmodular/EchoFXXX#1]
I built your module and unfortunately it does not work. Sound is amplified, but not delayed at all, the LM13700 gets hot after a few seconds.
While trying to figure out the error I found that in your foto of the board one of the 2N3906-transistor (the left one) is reversed. On the gerbers (and the boards I have obviously) it is the other way round. Might this be the problem?

**Comment by TOILmodular** [comment]
I do not think that your problem is related to the transistor. You are right, its orientation is different in the photo, well spotted. That was actually an error in my previous version, so I corrected it. The transistor needs to be in fact the other way round. Strangely, with the wrong position shown in the photo, the module still worked. It only made a strange noise with the gain knob set to a specific position, otherwise no issue. However, with the transistor in the right position (flat side to the left) it works fine. Your issue sounds to me more like some incorrect power supply to the 13700, since it is getting hot. Have you checked all ground and power connections to the IC? Maybe some short? Are you using the "original" or "Thonk" version?

**Comment by CoogLFish** [comment]
Thank you for the quick reply. My understanding of electronics is somewhat limited, but I suspected the transistor is not the major problem. My module doesn't work either way.
I'll check the board again for shorts and reflow the solder joints. I use the Thonk-version of the board.

**Comment by TOILmodular** [comment]
Ok. I did not build the "Thonk" version myself ro check if it works. But that version uses the same design for the main PCB, only the control PCBs are different.
However, the devil is in the details. Maybe there is still an issue which I did not realize.
If you want to take the risk of frying up another 13700, you might disconnect the control PCB from the main PCB and just power up the main one, checking if the 13700 still gets hot.
If so, we know that there is an issue with your main PCB, and that this is not due to the design, but rather an issue with your build.
Since the module works for me.
(Stupid question: Did you put in the IC the right way round?).
If the issue does not occur with the control board attached, then I should re-check the "Thonk" version.
I will take another look anyway.
Just let me know if you have any finding or any question.

**Comment by CoogLFish** [comment]
Thank you for taking the time, I really appreciate it. I checked all solder joints, everything seems to be fine. I'll check again with the boards disconnected, I have a few LM13700s to spare. I'll get back to you tonight.

**Comment by CoogLFish** [comment]
Oh well, also with disconnected boards the LM13700 heats up quickly. The troubleshooting continues ...

**Comment by TOILmodular** [comment]
I suggest you do some continuity tests at the following IC pins if you have a multimeter.
Pin 6 needs to be connected to -12V.
Pin 11 should be connected to +12V.
Both should not be connected to each other and to pins 4 and 13.
Those have to be connected to ground.

**Comment by CoogLFish** [comment]
Now this is strange and a little embarrassing. I did some continuity-tests and measured voltages on the ICs, everything was normal. Then I replaced the LM13700 with a cheap Chinese-knockoff I found somewhere in my drawers. What can I say? It worked! The supposedly real ones blew right away. Who would have thought of that ...
Thank you so much for taking the time for helping with the troubleshoot and of course for your design. I owe you one, big time!
